core/src/main/java/bio/terra/pearl/core/model/dataimport/Import.java

```diff
@@ -0,0 +1,23 @@
+package bio.terra.pearl.core.model.dataimport;
+
+import bio.terra.pearl.core.model.BaseEntity;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Getter @Setter @SuperBuilder
+@NoArgsConstructor
+public class Import extends BaseEntity {
+    private UUID responsibleUserId;
+    private UUID studyEnvironmentId;
+    private String type;
```

Comment:
best practice is to avoid using SQL-reserved words as column names.  (my fault for having this be the name in the design doc)
```suggestion
    private String importType;
```